errors.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Error responses
+---
+
+When error happens during request processing, the server responses HTTP status code of the error and the body of response cointains error description in the JSON format.
```

Comment:
If an error occurs during the handling of an API request, the server will respond with an adequate HTTP status code alongside a JSON body describing the error in more detail.